src/soupBowl.ts

```diff
@@ -37,7 +37,10 @@ class SoupBowl {
      * @returns {Soup.Message} Crafted message
      */
     newGetMessage(uri: string): Soup.Message {
-        return Soup.Message.new('GET', uri);
+        const message = Soup.Message.new('GET', uri);
+        // set User-Agent to appear more like a standard browser
+        message.request_headers.append('User-Agent', 'Mozilla/5.0');
```

Comment:
Is a random user agent string a good idea? While it should work, I expect it to cause issues with one or the other service/API. 

If you are certain that a random user agent will be as compatible as this one I can change it.

Reply:
I am not certain

Reply:
Then, let's leave it as it is for now.

Reply:
I tested various sites and actually found one that is protected by Cloudflare which rejects the User-Agent `Mozilla/5.0` because of lacking JS to complete the bot challenge. But `RandomWallpaperGnome3/3.0` as a User-Agent works and I don't get such a JS bot challenge.

* Not working:
  `http "https://danbooru.donmai.us/posts.json?tags=rating:general" User-Agent:Mozilla/5.0`
* Working:
  `http "https://danbooru.donmai.us/posts.json?tags=rating:general" User-Agent:RandomWallpaperGnome3/3.0`

We may want to expose this setting in the adapter to fill in. What do you think?